.github/workflows/ci.yml

```
@@ -23,16 +23,22 @@ jobs:
+          cache: pip
+          cache-dependency-path: pyproject.toml
+
+
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip wheel setuptools
-          pip install -e ".[crystal_space,dev]"
-          pip install pytest-cov
+          pip install uv
+          uv pip install -e ".[optional,dev]" --system
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)